### PR TITLE
Issue-54: Support for RequestMethod PATCH

### PR DESCRIPTION
**Pull request by jyothikomm**
Support for RequestMethod PATCH

### CODE REVIEW

File: src/main/java/com/edmunds/rest/databricks/restclient/AbstractDatabricksRestClientImpl.java [hunk 1]
```diff
@@ -128,6 +133,18 @@ protected HttpGet makeGetMethod(String path, Map<String, Object> data) {
     return method;
   }
 
+  protected HttpPatch makePatchMethod(String path, Map<String, Object> data)
```
Comment (samshuster) [inline src/main/java/com/edmunds/rest/databricks/restclient/AbstractDatabricksRestClientImpl.java]: what is the patch method used for in databricks api?

**Comment by samshuster** [comment]
apologies for being very late getting to this pr.
there appears to have been a problem with the repo owners getting notified which we will work on.

File: src/main/java/com/edmunds/rest/databricks/restclient/AbstractDatabricksRestClientImpl.java (hunk 1 above)
Comment (javamonkey79) [reply src/main/java/com/edmunds/rest/databricks/restclient/AbstractDatabricksRestClientImpl.java]: From what I can see, this is it @samshuster https://docs.databricks.com/dev-tools/api/latest/scim.html#update-user-by-id-patch

Comment (samshuster) [reply src/main/java/com/edmunds/rest/databricks/restclient/AbstractDatabricksRestClientImpl.java]: got it thanks @javamonkey79

**Pull request merged**